# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1335987783 -7200
#      Wed May 02 21:43:03 2012 +0200
# Branch gui
# Node ID 12b60f3b1394c6f4800e49e2a410bca2cc2b0a84
# Parent  8e9823066dad4a6d894acd623403bf5fd3e80a12
libqterminal: Updated to newest version.

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,3 +1,3 @@
 f9813bce2c06a6130a68db4478d1b16ddadaf276 gnulib
 34c2a274a1b607d2616d9903099a0905237c8f80 gui/qirc
-75e8afc065a08cbf8400dcb3386c5dc030cf8719 gui/qterminal
+a81f99b9b77d9a402b92463b41c5fdb82324349a gui/qterminal
